docs/concepts/storage/persistent-volumes.md

```diff
@@ -573,68 +573,53 @@ parameters:
 
 #### vSphere
 
-1. Create a persistent volume with a user specified disk format.
+   1. Create a persistent volume with a user specified disk format.
 
-```yaml
-kind: StorageClass
-apiVersion: storage.k8s.io/v1
-metadata:
-  name: fast
-provisioner: kubernetes.io/vsphere-volume
-parameters:
-  diskformat: zeroedthick
-```
+       ```yaml
+       kind: StorageClass
+       apiVersion: storage.k8s.io/v1
+       metadata:
+         name: fast
+       provisioner: kubernetes.io/vsphere-volume
+       parameters:
+         diskformat: zeroedthick
+       ```
 
--   `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
+       - `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
 
-2. Create a persistent volume with a disk format on a user specified datastore.
+    2. Create a persistent volume with a disk format on a user specified datastore.
```

Comment:
Should be "Create a StorageClass"?

Reply:
Fixed!

Reply:
Does this second example also encompass the 1st?  Can you combine the two?

Reply:
Also this second example should be "Create a StorageClass" too